src/files-card-body.tsx

```diff
@@ -155,10 +157,12 @@ export const FilesCardBody = ({
     clipboard: string[], setClipboard: React.Dispatch<React.SetStateAction<string[]>>,
     showHidden: boolean,
     setShowHidden: React.Dispatch<React.SetStateAction<boolean>>,
+    setDragDropActive: React.Dispatch<React.SetStateAction<boolean>>,
 }) => {
     const [boxPerRow, setBoxPerRow] = useState(0);
     const dialogs = useDialogs();
     const { addAlert, cwdInfo } = useFilesContext();
+    const { uploadedFiles } = useContext(UploadContext);
```

Comment:
Why? It is not used here? Nor is anything else relevant to Upload, it just gets passed down.